config/environment.js

```diff
@@ -132,6 +132,7 @@ module.exports = function(environment) {
             devMode,
             statusCookie,
             cookieDomain,
+            secondaryNavbarId: 'secondaryNavbar',
```

Comment:
Two things:
1. I don't think this belongs here as it's not something we would set with an environment variable. It should probably be somewhere else in config instead.
2. Anything added to `config/environment.js` needs to be typed in `config/environment.d.ts`